cmd/agent/app/reporter/tchannel/reporter.go

```diff
@@ -131,6 +131,8 @@ func (r *Reporter) submitAndReport(submissionFunc func(ctx thrift.Context) error
 		r.logger.Error(errMsg, zap.Error(err))
 		return err
 	}
+
+	r.logger.Debug("Span batch submitted by the agent", zap.Int64("batch-size", size))
```

Comment:
nit: batch-size is a  bit misleading, could be interpreted as size in bytes. `span-count` would be unambiguous.